src/main/java/uk/gov/hmcts/darts/common/repository/ArmAutomatedTaskRepository.java

```diff
@@ -4,6 +4,11 @@
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.darts.common.entity.ArmAutomatedTaskEntity;
 
+import java.util.Optional;
+
 @Repository
 public interface ArmAutomatedTaskRepository extends JpaRepository<ArmAutomatedTaskEntity, Integer> {
+
+    Optional<ArmAutomatedTaskEntity> findByAutomatedTask_taskName(String taskName);
```

Comment:
Was there a change that mean't we were supposed to write integration tests for new repository methods?

Reply:
Indeed, good spot. Have added some integration tests in this area.